Justfile

```diff
@@ -53,7 +68,7 @@ dev: pnpm_install
   trunk serve --port="{{trunk_port}}" --open
 
 # Run all application regression tests
-test-e2e: pnpm_install
+test-e2e: pnpm_install deploy-mina-indexer && shutdown-mina-indexer
```

Comment:
question(non-blocking): When should you `&&` and when should you just space? Is there a difference?

Reply:
Yes, && runs after, space runs before.

Reply:
[Here](https://just.systems/man/en/dependencies.html#running-recipes-at-the-end-of-a-recipe) is the documentation